sdk/storage/azure-storage-blob/tests/test_blob_client.py

```diff
@@ -473,16 +473,15 @@ def callback(response):
         exists = container.get_container_properties(raw_response_hook=callback)
         self.assertTrue(exists)
 
+    @pytest.mark.live_test_only
     @GlobalStorageAccountPreparer()
     def test_client_request_id_echo(self, resource_group, location, storage_account, storage_account_key):
         # client request id is different for every request, so it will never match the recorded one
         pytest.skip("Issue tracked here: https://github.com/Azure/azure-sdk-for-python/issues/8098")
```

Comment:
Is there a way to skip the test in a decorator?
Otherwise it will be doing the account setup even though it's going to be skipped anyway?